ihmc-high-level-behaviors/src/main/java/us/ihmc/PerceptionAndAutonomyProcess.java

```diff
@@ -644,12 +682,16 @@ public static void main(String[] args)
       ROS2Node ros2Node = ROS2Tools.createROS2Node(CommunicationMode.INTERPROCESS.getPubSubImplementation(), "perception_autonomy_process");
       ROS2Helper ros2Helper = new ROS2Helper(ros2Node);
 
+      RapidHeightMapExtractor.getHeightMapParameters().setCropWindowSize(501);
```

Comment:
Okay actually I think this is a better strategy where the parameters don't have to get duplicated.